packages/google_maps_flutter/google_maps_flutter/example/lib/marker_icons.dart

```diff
@@ -29,8 +29,8 @@ class MarkerIconsBody extends StatefulWidget {
 const LatLng _kMapCenter = LatLng(52.4478, -3.5402);
 
 class MarkerIconsBodyState extends State<MarkerIconsBody> {
-  GoogleMapController controller;
-  BitmapDescriptor _markerIcon;
+  GoogleMapController? controller;
+  BitmapDescriptor? _markerIcon;
```

Comment:
ditto

Reply:
There are flows in the existing logic that assume _markerIcon can be null.